docs/samples/box/shadow.md

```diff
@@ -0,0 +1,169 @@
+# Shadow
+
+```js chart-editor
+// <block:setup:4>
+const DATA_COUNT = 8;
+const MIN = [0, 50];
+const MAX = [50, 100];
+
+Utils.srand(8);
+
+const data = {
+  datasets: [{
+    data: Utils.points({count: DATA_COUNT, min: MIN[0], max: MAX[0]})
+  }, {
+    data: Utils.points({count: DATA_COUNT, min: MIN[1], max: MAX[1]})
+  }]
+};
+// </block:setup>
+
+// <block:annotation1:1>
+const annotation1 = {
+  type: 'box',
+  backgroundColor: 'rgba(0,150,0,0.02)',
+  borderColor: 'rgb(0,150,0)',
+  borderRadius: {
+    topLeft: 4,
+    topRight: 4,
+    bottomLeft: 4,
+    bottomRight: 4
+  },
+  borderWidth: 1,
+  xMax: (ctx) => max(ctx, 0, 'x') + 2,
+  xMin: (ctx) => min(ctx, 0, 'x') - 2,
+  yMax: (ctx) => max(ctx, 0, 'y') + 2,
+  yMin: (ctx) => min(ctx, 0, 'y') - 2,
+  beforeDraw: (ctx) => drawShadow(ctx)
+};
+// </block:annotation1>
+
+// <block:annotation2:2>
+const annotation2 = {
+  type: 'box',
+  backgroundColor: 'rgba(150,0,0,0.02)',
```

Comment:
Same as above

Reply:
@LeeLenaleee do you mean inside of color definitions?

Reply:
done|